src/librustdoc/html/render/mod.rs

```diff
@@ -3682,8 +3682,9 @@ fn spotlight_decl(decl: &clean::FnDecl, cache: &Cache) -> String {
         if let Some(impls) = cache.impls.get(&did) {
             for i in impls {
                 let impl_ = i.inner_impl();
-                if impl_.trait_.def_id_full(cache).map_or(false, |d| cache.traits[&d].is_spotlight)
-                {
+                if impl_.trait_.def_id().map_or(false, |d| {
+                    cache.traits.get(&d).map(|t| t.is_spotlight).unwrap_or(false)
```

Comment:
Why was this changed from panicking to defaulting to `false`? Is this unrelated to the `is_spotlight` field removal?

Reply:
Both code do the same thing as far as I can tell. Just in more "convoluted" ways.

Reply:
I don't think so. Before it would panic if `d` did not exist in `cache.traits`. Now it just doesn't run the branch. The new behavior is probably better, but I want to make sure it's intentional.

Reply:
It is. :)